### PR TITLE
September release notes

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_help/release_notes/2023/9_19_23.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+## Commenting in Canvas
+
+[Comments in Canvases]({{site.baseurl}}/user_guide/engagement_tools/canvas/create_a_canvas/canvas_comments/) can be a great collaborative space for your marketing team to identify, discuss, and review the finer details of your Canvases. As you build out a Canvas, you can make and manage comments to identify these areas that may require additional feedback from your colleagues.
+
+## Deliverability Center
+
+The [Deliverability Center]({{site.baseurl}}/user_guide/data_and_analytics/analytics/deliverability_center) provides more insight into your email performance by supporting the use of Gmail Postmaster Tools to track data on emails sent and gather data about your sending domain. 
+
+Email deliverability is the core of campaign success. Using the Deliverability Center in the Braze dashboard, you can view your domains by IP Reputation or Delivery Errors to discover and troubleshoot any potential issues with email deliverability.
```
Comment (lydia-xie) [inline _docs/_help/release_notes/2023/9_19_23.md:27]:
```suggestion
Email deliverability is the core of campaign success. Using the Deliverability Center in the Braze dashboard, you can view your domains by IP reputation or delivery errors to discover and troubleshoot any potential issues with email deliverability.
```

**Comment by rachel-feinberg** [comment]
@bre-fitzgerald Will a tile be added here later: https://braze-docs-h7q5upoau-braze.vercel.app/docs/help/release_notes/2023?

File: _docs/_help/release_notes/2023/9_19_23.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+The Braze and [Ketch]({{site.baseurl}}/partners/data_and_infrastructure_agility/customer_data_platform/ketch/) integration allows you to control customer communication preferences within the Ketch preference center and automatically propagate these changes to Braze.
+
+### Redpoint – Customer Data Platform
+
+Redpoint is a technology platform that provides marketers with a fully integrated campaign orchestration platform. The Braze and [Redpoint]({{site.baseurl}}/partners/data_and_infrastructure_agility/customer_data_platform/redpoint/) integration allows you to create Braze segments based on your Redpoint CDP data. 
+
+### Simon Data – Customer Data Platform
+ 
+Use the Braze and [Simon Data]({{site.baseurl}}/partners/data_and_infrastructure_agility/customer_data_platform/simondata/) integration to create and sync sophisticated audiences to Braze for orchestration in real-time and without code. With this integration, you can leverage the best of Simon’s campaign prioritization and Identity matching capabilities, complex aggregate support, and more to elevate your Braze campaigns downstream.
```
Comment (rachel-feinberg) [inline _docs/_help/release_notes/2023/9_19_23.md:90]:
```suggestion
Use the Braze and [Simon Data]({{site.baseurl}}/partners/data_and_infrastructure_agility/customer_data_platform/simondata/) integration to create and sync sophisticated audiences to Braze for orchestration in real-time and without code. With this integration, you can leverage the best of Simon’s campaign prioritization and identity-matching capabilities, complex aggregate support, and more to elevate your Braze campaigns downstream.
```

From the doc, it doesn't look like "Identity" should be capitalized.

**Review by rachel-feinberg — APPROVED** [review]
Lgtm! Just one suggestion

File: _docs/_help/release_notes/2023/9_19_23.md (hunk 2 above)
Comment (bre-fitzgerald) [reply _docs/_help/release_notes/2023/9_19_23.md:90]: Thanks for catching!

**Comment by bre-fitzgerald** [comment]
> @bre-fitzgerald Will a tile be added here later: [braze-docs-h7q5upoau-braze.vercel.app/docs/help/release_notes/2023](https://braze-docs-h7q5upoau-braze.vercel.app/docs/help/release_notes/2023)?

Yes ma'am! And to the main release notes page 👍

**Pull request merged**